taf/tests/test_repository_tool.py

```diff
@@ -144,10 +120,11 @@ def test_update_targets_valid_key_valid_pin(taf_happy_path, targets_yk):
         "dummy/target_dummy_repo": {"target": {"commit": target_commit_sha}},
         "capstone": {},
     }
-
+    yk.add_key_pin(targets_yk.serial, "123456")
```

Comment:
Could you please replace all `"123456"` strings with `yubikey_utils.VALID_PIN` contant?